pom.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.jenkins-ci.plugins</groupId>
+    <artifactId>plugin</artifactId>
+    <version>2.12</version>
+    <relativePath/>
+  </parent>
+  <groupId>org.jenkins-ci.plugins</groupId>
+  <artifactId>help-editor</artifactId>
+  <version>1.0-SNAPSHOT</version>
+  <packaging>hpi</packaging>
+
+  <properties>
+    <jenkins.version>1.625.3</jenkins.version>
+    <java.level>7</java.level>
+  </properties>
+
+  <name>Help editor</name>
+  <description>Help editor plugin allows you to add additional custom help above default help on job configuration pages.</description>
+  <url>https://wiki.jenkins-ci.org/display/JENKINS/Help+Editor+Plugin</url>
```

Comment:
It will look it up once it is available in the update center, so this is fine as it is. 

The wiki page could do with a link or copy of the documentation from the README